src/routes/jira/jira-get.ts

```diff
@@ -160,13 +161,15 @@ const renderJiraCloudAndEnterpriseServer = async (res: Response, req: Request):
 
 	const hasConnections =  !!(installations.total || gheServers?.length);
 
+	const useNewSPAExperience = await booleanFlag(BooleanFlags.USE_NEW_5KU_SPA_EXPERIENCE, jiraHost);
```

Comment:
What if we change the URL in the `atlassian-connect.json` instead of this?
Cause this firstly loads all the info for the current GitHub for Jira app, then does a redirect. We don't need to load anything from the current app.

Reply:
Actually it doesn't `then does a redirect`
This page is the home page, if the user click the left sidebar, it will goes to home page. So I still want user able to go to home page at some point.